packages/venia-ui/lib/components/CartPage/PriceAdjustments/ShippingMethods/shippingForm.css

```
@@ -27,6 +27,10 @@
     padding: 0.75rem 1rem;
 }
 
+.error {
+    color: rgb(var(--venia-error));
```

I'm not sure this is a property anymore.

- See [`tokens.css`](https://github.com/magento/pwa-studio/blob/develop/packages/venia-ui/lib/tokens.css#L44) for new tokens
- If you replace this one, make sure to replace all of the others as well

```suggestion
    color: rgb(var(--venia-global-color-error));
```

---

@jimbo - that token doesn't seem to exist. Would you like me to add it, or pick one of the `--venia-global-color-red-xxx` values?

---

Strike that, we cut the release branch before design tokens landed, so the new tokens are not available. Going to leave this as-is, and we'll resolve when we merge release back into `develop`.

---

Ah, my mistake. I assumed this was targeting `develop` but it's actually targeting the release branch. Carry on. 👍